potodo/interactive.py

```diff
@@ -80,6 +92,9 @@ def get_dir_list(
 def get_files_from_dir(
     directory: str, repo_path: Path, exclude: Iterable[Path]
 ) -> List[str]:
+    """
+    Return a list of directory.
```

Comment:
This function returns a list of `.po` files from the given path + dir, excluding the files in the excluded list